RequestManagementSystem/Client/Request.py

```diff
@@ -534,10 +534,13 @@ def optimize( self ):
     repAndRegList = []
     removeRepList = []
     i = 0
-    while i < len( self.__operations__ ) - 1:
```

Comment:
I can't remember why I did that but there must have been a good reason... will comment when fixing the code ;-)